scripts/ssh_bosh.sh

```diff
@@ -1,14 +1,14 @@
 #!/bin/bash
 set -euo pipefail
 
-BOSH_KEY=/tmp/bosh_id_rsa.$RANDOM
+BOSH_KEY=/tmp/id_rsa.$RANDOM
 BOSH_IP=$(aws ec2 describe-instances \
-    --filters "Name=key-name,Values=${DEPLOY_ENV}_bosh_ssh_key_pair" \
+    --filters 'Name=tag:Name,Values=bosh/*' "Name=key-name,Values=${DEPLOY_ENV}_key_pair" \
```

Comment:
We could use these filters instead: `"Name=tag:deploy_env,Values=${DEPLOY_ENV}" 'Name=tag:job,Values=bosh' `